modules/effects/arrow-effects-reactor-instances/src/main/kotlin/arrow/effects/FluxKInstances.kt

```diff
@@ -115,7 +115,7 @@ interface FluxKAsyncInstance :
   Async<ForFluxK>,
   FluxKMonadDeferInstance {
   override fun <A> async(fa: Proc<A>): FluxK<A> =
-    FluxK.runAsync(fa)
+    FluxK.runAsync { _, cb -> fa(cb) }
```

Comment:
same here, what's the intention on exposing it to callers if it's ignored inside ?